ical-relay/config/config.toml

```diff
@@ -0,0 +1,13 @@
+[server]
+addr = ":8080"
+loglevel = "debug"
+
+[profiles]
+  [profiles.du4lcombined]
+    url = "http://api:3000/calendar.ics?key=27b4baf492e1dadd5f525e4ca04dbaf8"
+    regex = [""]
+    public = true
+    from = "2021-12-02T00:00:00Z"
+    until = "2021-12-31T00:00:00Z"
+    passid = true
+    addurl = ["https://outlook.live.com/owa/calendar/00000000-0000-0000-0000-000000000000/d8c3ea1e-74f4-4920-8b55-6166e4c9f57e/cid-DB8463EC38A495B6/calendar.ics"]
```

Comment:
I think this config should just be a "template" with no specific strings configured so that whoever hosts it can configure themselves, like the .env files.

Since we're not exposing api:3000/calendar.ics publicly right now anyway, maybe we should just drop the key or at least set it to disabled by default? That way the config can just always be http://api:3000/calendar.ics and the only addurl parameter needs to be configured

Reply:
I'd be in favor of dropping the key

And thats what the todo part was kind of about. I don't think we should keep the config in this repo, but do we have another system for managing those configs?